draft-ietf-taps-interface.md

```diff
@@ -937,9 +937,8 @@ Type:
 This property specifies that a message should be sent in such a way
 that the transport protocol ensures all data is received on the other side
 without corruption. Changing the ´Reliable Data Transfer´ property on Messages
-is only possible if the transport protocol supports
-partial reliability (see {{prop-partially-reliable}}).
-Therefore, for protocols that always transfer data reliably, this property is always true and for protocols that always transfer data unreliably, this property is always false. Changing it may generate an error.
+is only possible if the Connection supports reliability.
+When this is not the case, changing it will generate an error.
```

Comment:
Agree that if the Connection does not support reliability and the application requests reliable, we must throw an error so that sending fails, but we don't need to fail the other way round. Maybe a warning makes sense, but we don't need to standardize that.
However, I think this property makes most sense with partially reliable transport protocols, and I'd like to keep a reference to {{prop-partially-reliable}} here. Maybe something like "if the Connection supports reliability, see {{prop-reliable}}, or partial reliability, see {{prop-partially-reliable}}"?

Reply:
For me, this is clear.

Reply:
... My choice would be I'd prefer to standardise "needs reliable", if I had a choice of semantics.
Relaxing "needs reliable" can be done on a per-message basis for SCTP stacks that allow this.

Reply:
I see -- I misunderstood the intention behind "Reliable Data Transfer (Connection)". I suggest to include the reference to partially reliability as @theri suggests, but also update the text after "Reliable Data Transfer (Connection)" from: "This property specifies whether the application wishes to use a transport protocol that ensures that all data is received on the other side without corruption."  to "This property specifies whether the application requires that all data it sends on the Connection will be received on the other side without corruption."

The main point here is the use of "requires" as opposed to "wishes".

Reply:
Well, "Reliable Data Transfer (Connection)" has the data type "Preference" anyway, so an application can Require, Prefer, Avoid, or Prohibit it. Out of these, I guess "Require" makes the most sense, and I think it can replace "wishes", which sounds like "Prefers".
However, we have lots and lots of formulations like this in the other properties as well, so maybe we want to agree on a more general non-awkward way to phrase this (in another PR?).

Reply:
Ah. Understood.  But let me just call it "needs" for now.